src/core/utilities.scss

```diff
@@ -167,23 +238,95 @@
   text-align: justify;
 }
 
+// Line Height
+
+.line-height-none {
```

Comment:
Adding these goes against a core concept of separation of concerns - keeping style separated from markup. We already break this concept by allowing for layout details as utilities within markup that I'd rather not do for many reasons but I understand for spiking layout. Style should remain separated from markup and not be used in this way. Doing this will encourage the overuse and potential abuse of utility classes which is not a direction we want to go in. (ie tailwind).